packaging/ondemand.spec

```diff
@@ -188,6 +188,8 @@ touch %{buildroot}%{_sysconfdir}/ood/config/ood_portal.sha256sum
 Defaults:apache !requiretty, !authenticate
 Defaults:apache env_keep += "NGINX_STAGE_* OOD_*"
 apache ALL=(ALL) NOPASSWD: /opt/ood/nginx_stage/sbin/nginx_stage
+Cmnd_Alias KUBECTL = /usr/local/bin/kubectl, /usr/bin/kubectl, /bin/kubectl
```

Comment:
Was this like a cherry-pick from master or something? This change was made a while ago.

Reply:
Yep, it was committed accidentally. Obviously I'm using it as reference point and checked out the master version and accidentally committed it.